hashing/src/chunk_with_proof.rs

```diff
@@ -0,0 +1,322 @@
+use crate::indexed_merkle_proof::IndexedMerkleProof;
+
+use casper_types::{
+    allocate_buffer,
+    bytesrepr::{Bytes, FromBytes, ToBytes},
+};
+
+use crate::{error, Digest};
```

Comment:
We should merge imports where we can, and bring types fully into scope generally.

```suggestion
use schemars::JsonSchema;
use serde::{Deserialize, Serialize};

use casper_types::bytesrepr::{self, Bytes, FromBytes, ToBytes};

use crate::{error, indexed_merkle_proof::IndexedMerkleProof, Digest};
```

Reply:
Imports have been cleaned up. Here and in other files.